### DISCUSSION TITLE
Possible bug: Tabs incorrect behavior (filling widget space) when combined with window menus

### DISCUSSION THREAD

**Discussion by TheQuantumPhysicist** [hecrj/iced#1819] [Q&A]
Hello

I combined the [menu](https://github.com/iced-rs/iced_aw/tree/main/examples/menu) example with the [tabs](https://github.com/iced-rs/iced_aw/tree/main/examples/tabs) example, but the result seems to be faulty somehow. The tabs bar fills the whole screen no matter what. I tried to change everything possible (even removing the icons altogether), but nothing helped. 

**Please find a minimal reproducible example in this repo**: https://github.com/TheQuantumPhysicist/IcedTabsTest

build and run with `cargo run`. On some of my machines I needed to install `libfontconfig-dev`.

It may be a dumb mistake of mine, and if it's, I apologize. I'm new to this library. But from what I'm seeing, all I did was move the code from example to here, remove some variants and change some labels. 

Thank you and have a great day.

**Comment by hecrj** [comment]
Please, open an issue in `iced_aw`: https://github.com/iced-rs/iced_aw/issues

**Comment by TheQuantumPhysicist** [comment]
[Done](https://github.com/iced-rs/iced_aw/issues/118).
